src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php

```diff
@@ -6,6 +6,11 @@
 use Swarrot\Processor\ProcessorInterface;
 use Psr\Log\LoggerInterface;
 
+/**
+ * Class ExceptionCatcherProcessor
+ *
+ * @package Swarrot\Processor\ExceptionCatcher
```

Comment:
There is definitively a lack of documentation regarding swarrot. I need to take time to write some & add contributing information. :/
As said by @stof, we don't use the `@package` annotation in swarrot and the doc block you add on this class doesn't not provide any value, can you remove it?
Thanks for your contribution. :)